src/sections/newcontactuspage/contactUs/ContactUs.css

```diff
@@ -0,0 +1,146 @@
+.backgroundforcontactus {
+    background-color: #EDF3F5;
+    min-height: 100vh; /* Ensures the background covers full height even if content is short */
+    font-family: "Jura", sans-serif;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 0 20px; /* Adds padding to prevent content from touching the screen edges */
+}
+
+.division {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 120px;
+}
+
+.secondcolumnnn {
+    display: flex;
+    flex-direction: column;
+    gap: 30px;
+}
+
+#contactus {
+    font-size: 64px;
+    font-weight:Normal;
+    
+}
+
+#location {
+    font-size: 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    
+}
+
+#address {
+    font-size: 24px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.iconlist{
+    display:flex;
+    flex-direction: column; 
+    gap: 25px;
+    justify-content: center;
+    align-items: center;
+
+
+
+}
+
+.iconlist li{
+    font-size: 1.2rem;
+}
+
+.iconlist li a {
+    color: black; /* Set the color to black */
+    text-decoration: none; /* Remove the underline */
+  }
+  
+  .iconlist li a:hover {
+    color: white; /* Ensure it stays black on hover */
+  }
+  
+/* Responsive for tablets and smaller screens */
+@media (max-width: 768px) {
+    .background {
+        padding: 0 15px; /* Adds some space on tablet devices */
+    }
+
+    .division {
+        flex-direction: column;
+        gap: 50px; /* Reduced gap for better layout on smaller screens */
+    }
+
+    #contactus {
+        font-size: 48px;
+        text-align: center;
+    }
+
+    #location, #address {
+        font-size: 18px;
+        text-align: center; /* Center the text for better alignment */
+    }
+    .iconlist {
+        flex-direction: row; /* Switch to row layout for larger screens */
+        gap: 2rem; /* Adjust gap size for larger screens */
+        text-align: center;
+        justify-items: center;
+    }
+}
+
+/* Responsive for smaller mobile screens */
+@media (max-width: 540px) {
```

Comment:
Sorry, I requested a change to this in the last review. That was a mistake, change this back to `480px`. The change was actually intended for another file mentioned in this review.